instance/models/mixins/openedx_config.py

```diff
@@ -311,3 +314,48 @@ def _get_configuration_variables(self):
             })
 
         return template
+
+    def _get_prometheus_variables(self):
+        """Get all Prometheus-related ansible variables specific to Open edX appservers."""
+        return {
+            **super()._get_prometheus_variables(),
+            'NODE_EXPORTER_NGINX_CONFIG_PATH': '/edx/app/nginx/sites-available/node-exporter',
+            'NODE_EXPORTER_HTPASSWD_PATH': '/edx/app/nginx/nginx.htpasswd',
+            'NODE_EXPORTER_USE_SSL': False,
+        }
+
+    def _get_consul_variables(self):
+        """Get all Consul-related ansible variables specific to Open edX appservers."""
+        return {
+            **super()._get_consul_variables(),
+            'consul_service_config': {
+                'service': [
+                    {
+                        "id": self.server_hostname,
+                        "name": "edxapp",
```

Comment:
I noticed that we are using the same service name for all Open edX instances, which I don't think we should.  I think all appservers of an Open edX instance should use the same service name, but different Open edX instances provide different services.  For this reason, I propose to change this name to `"edxapp-{}".format(self.id)"`.